Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/SpectatorViewPluginAPI.cs

```diff
@@ -15,14 +15,14 @@ namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.M
     /// </summary>
     public class SpectatorViewPluginAPI
     {
-#if UNITY_WSA
+#if WINDOWS_UWP
```

Comment:
>WINDOWS_UWP [](start = 4, length = 11)

When is this defined compared to UNITY_WSA?  The structure of this file actually confuses me a little bit. In the past when I've had an editor and a WSA version of a plugin, the plugin DLLs have had the same name with .meta files that correctly load them in editor versus UWP.  Maybe SpectatorViewPlugin.Editor should just be a different build of SpectatorViewPlugin with appropriate .meta information?

Reply:
Yeah, i'm not sure the best practice for maintaining this. We could definitely have these plugins built with the same name. But they are built/configured through different visual studio projects because one is uwp friendly while the other is not.

With this specific fix, UNITY_WSA seems to be enabled when runniing things in the editor when set to build for UWP. WINDOWS_UWP is only picked up when generating the actual uwp visual studio solution. So, this fix will make it easier to not get dll missing exceptions in the editor based on build configurations. If its felt that we should name these dlls the same, we can do that, but i'd prefer to check this in now and fix that later. It may be more practical to fix the dll naming when all of this code is in a shared repo.